### ISSUE TITLE
Does libde265 support scc (Screen Content Coding)?

### ISSUE THREAD

**Issue by open-trade** [strukturag/libde265#272]
I see hmscc encoder in tools/rd-curves

**Comment by farindk** [comment]
No. The scc config has only been used to check the potential performance gains. It was not implemented in the decoder.
Do you know anyone using this in practice?

**Comment by open-trade** [comment]
i am going to use scc in my remote desktop software.

**Comment by farindk** [comment]
How is your remote desktop software? Are there any SCC streams for testing?

**Comment by ValZapod** [comment]
https://github.com/FFmpeg/FFmpeg/commit/c4ae6908f28ceacb39b96de9f8761aae1968bf42

And https://patchwork.ffmpeg.org/project/ffmpeg/list/?series=2878 foe hevc decoder in  ffmpeg.

**Comment by farindk** [comment]
x265 now supports SCC and MV. Might be worth taking a look to generate some test streams.
